lib/connect.c

```diff
@@ -1294,7 +1294,29 @@ static CURLcode cf_setup_connect(struct Curl_cfilter *cf,
   /* sub-chain connected, do we need to add more? */
 #ifndef CURL_DISABLE_PROXY
   if(ctx->state < CF_SETUP_CNNCT_SOCKS && cf->conn->bits.socksproxy) {
-    result = Curl_cf_socks_proxy_insert_after(cf, data);
+    /* Where to we need the SOCKS to connect to? */
```

Comment:
```suggestion
    /* Where do we need the SOCKS to connect to? */
```

?

Reply:
Thanks. Fixed.

Reply:
I don't see the fix?

Reply:
Because I had pushed before the commit. Sorry.